games-board/cockatrice/cockatrice-20230914.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg cmake
+
+MY_PV="2023-09-14-Release-2.9.0"
+
+DESCRIPTION="Open-source multiplatform software for playing card games over a network"
+HOMEPAGE="https://github.com/Cockatrice/Cockatrice"
+SRC_URI="https://codeload.github.com/Cockatrice/Cockatrice/tar.gz/refs/tags/${MY_PV} -> ${P}.tar.gz"
+S="${WORKDIR}/Cockatrice-${MY_PV}"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+client +oracle test server"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-qt/qtconcurrent:5
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5[ssl]
+	dev-qt/qtprintsupport:5
+	dev-qt/qtwebsockets:5
+	dev-qt/qtwidgets:5
+	client? (
+		dev-libs/protobuf:=
+		dev-qt/qtmultimedia:5
+		dev-qt/qtsvg:5
+	)
+	oracle? ( sys-libs/zlib )
```

Comment:
https://github.com/Cockatrice/Cockatrice/blob/2023-09-14-Release-2.9.0/oracle/CMakeLists.txt#L69..L88

automagic dependency on both sys-libs/zlib and app-arch/xz-utils with use oracle

Reply:
I have added app-arch/xz-utils to the RDEPEND